build/tools/Automation/Python/PythonConstants.cs

```diff
@@ -19,6 +19,9 @@ public class PythonConstants
         public const string PythonSuffixUrl = "/python?restype=container&comp=list&include=metadata";
         public static readonly Dictionary<string, string> VersionGpgKeys = new Dictionary<string, string>()
         {
+            { "3.7", "E3FF2839C048B25C084DEBE9B26995E310250568" },
```

Comment:
Would this file be updated every time we add support for a new minor version?

Reply:
That's correct. There was no reliable and consistent way found of obtaining the new GPG keys.

Reply:
Sounds good! Thank you for clarifying!